.buildkite/commands/build-and-upload-to-testflight.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -eu
+
+echo "--- :rubygems: Setting up Ruby gems"
+install_gems
+
+echo "--- :node: Installing NPM Dependencies and Syncing Project"
+npm ci
+npm run build
+npx cap sync
+
+echo "--- :testflight: Build and upload to TestFlight"
+bundle exec fastlane build_and_upload_to_testflight
```

Comment:
Have you considered splitting build from upload in two steps?

I did this recently [in Pocket Casts iOS](https://github.com/Automattic/pocket-casts-ios/pull/1808) and I was pleased with the result. The advantage is that if the upload fails, we can retry without rebuilding the binary.